src/sentry/tsdb/redissnuba.py

```diff
@@ -59,7 +59,9 @@ def dont_do_this(callargs):
 
 model_backends = {
     # model: (read, write)
-    model: ("redis", "redis") if model not in SnubaTSDB.model_query_settings else ("snuba", "dummy")
+    model: ("redis", "redis")
+    if model not in SnubaTSDB.non_outcomes_query_settings
+    else ("snuba", "dummy")
```

Comment:
Isn't this going to fall back to redis for outcomes everywhere ?

Reply:
It won't for sentry.io prod because that uses the service delegator and not `RedisSnubaTSDB` (getsentry:conf/settings/prod.py#L131-L152).

It will for single tenant and on premise because they use `RedisSnubaTSDB`.

I guess it'll revert it back for sentry.io staging. But I can follow up this change with using ServiceDelegator for staging too (maybe dev as well?).